web/src/App.tsx

```diff
@@ -67,7 +67,7 @@ export default function App(): ReactElement {
   useEffect(() => {
     if (Capacitor.isNativePlatform() && Capacitor.getPlatform() === 'android') {
       Cap.addListener('backButton', () => {
-        if (window.location.pathname === '/map') {
+        if (window.location.pathname === '/map/24h') {
```

Comment:
The idea is this will exit the app if you press the back button from the landing page. I think if someone navigates to map/30d then it should take two back button presses to exit the app.